services/src/test/java/org/exoplatform/task/dao/TestTaskDAO.java

```diff
@@ -782,25 +782,27 @@ public void testGetWatchersOfTask() throws Exception{
   @Test
   public void testFindTasks() throws Exception {
```

Comment:
your test do not correctly test your modification : 
you should have at least a unit test doing this : 
user1 creates a task task1 in project1
user1 creates a task task2 in project2
user2 is participator of project1
user2 is not participator of project2
search task for user2 should return 1 one task : task1

Can you add a test with this scenario ?

Reply:
Done !!!